### DISCUSSION TITLE
GPTH or built-in?

### DISCUSSION THREAD

**Discussion by assafdori** [TheLastGimbus/GooglePhotosTakeoutHelper#332] [General]
Hey,

I am looking to transfer my photos from Google Photos to iCloud and came up across this repository. However, I also noticed that it's possible to migrate the photos directory from Google Photos to iCloud, as can be seen here: 

https://support.apple.com/en-us/HT213483

So, I'm wondering what benefit I can get using GPTH?

Thanks.

**Comment by TheLastGimbus** [comment]
Apple may have done a good job at this. Or they may not, I don't know

What i know is that `gpth` came through a shit ton of little edge cases ans weird quirks to be as stable as possible

However it is still not 100.00% bulletproof as it still has problem with for example motion photos

Will Apple be better than this? I don't know, i encurage you to try :+1:
